moveit_kinematics/kdl_kinematics_plugin/src/kdl_kinematics_parameters.yaml

```diff
@@ -0,0 +1,52 @@
+kdl_kinematics:
+  joints: {
+    type: string_array,
+    default_value: ["joint1", "joint2", "joint3"],
```

Comment:
I don't think we want a default for this. Maybe remove these defaults so the user has to specify these joints.